test/api_integration/apis/saved_objects/migrations.ts

```diff
@@ -1,763 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-/*
- * Smokescreen tests for core migration logic
- */
-
-import uuidv5 from 'uuid/v5';
-import { set } from '@elastic/safer-lodash-set';
-import _ from 'lodash';
-import expect from '@kbn/expect';
-import { SavedObjectsType } from 'src/core/server';
-import { Client as ElasticsearchClient } from '@elastic/elasticsearch';
-
-import {
-  DocumentMigrator,
-  IndexMigrator,
-  createMigrationEsClient,
-} from '../../../../src/core/server/saved_objects/migrations/core';
-import { SavedObjectsTypeMappingDefinitions } from '../../../../src/core/server/saved_objects/mappings';
-
-import {
-  SavedObjectsSerializer,
-  SavedObjectTypeRegistry,
-} from '../../../../src/core/server/saved_objects';
-import { FtrProviderContext } from '../../ftr_provider_context';
-
-const KIBANA_VERSION = '99.9.9';
-const FOO_TYPE: SavedObjectsType = {
-  name: 'foo',
-  hidden: false,
-  namespaceType: 'single',
-  mappings: { properties: {} },
-};
-const BAR_TYPE: SavedObjectsType = {
-  name: 'bar',
-  hidden: false,
-  namespaceType: 'single',
-  mappings: { properties: {} },
-};
-const BAZ_TYPE: SavedObjectsType = {
-  name: 'baz',
-  hidden: false,
-  namespaceType: 'single',
-  mappings: { properties: {} },
-};
-const FLEET_AGENT_EVENT_TYPE: SavedObjectsType = {
-  name: 'fleet-agent-event',
-  hidden: false,
-  namespaceType: 'single',
-  mappings: { properties: {} },
-};
-
-function getLogMock() {
-  return {
-    debug() {},
-    error() {},
-    fatal() {},
-    info() {},
-    log() {},
-    trace() {},
-    warn() {},
-    get: getLogMock,
-  };
-}
-export default ({ getService }: FtrProviderContext) => {
-  const esClient = getService('es');
-  const esDeleteAllIndices = getService('esDeleteAllIndices');
-
-  describe('Kibana index migration', () => {
-    before(() => esDeleteAllIndices('.migrate-*'));
-
-    it('Migrates an existing index that has never been migrated before', async () => {
-      const index = '.migration-a';
-      const originalDocs = [
```

Comment:
This must be a very old FTR test, as it's not really using the Kibana instance and was directly manipulating the `IndexMigrator` instead. As this class is no longer, I deleted the test suite, and as v2 is already properly covered in other FTR and IT tests suite, I did not add additional tests.